service/src/main/resources/application.yml

```diff
@@ -32,9 +32,9 @@ spring:
     password: ${springMailPassword:"none"}
     port: 465
     protocol: 'smtps'
-    dev-mode: true
+    devMode: true
     debug: true
-    test-connection: false
```

Comment:
spring.mail.test-connection is for sure the spring native property and the name cannot be changed to camel case